src/ssh/sshConfig.ts

```diff
@@ -73,6 +74,46 @@ export default class SSHConfiguration {
         return new SSHConfiguration(config);
     }
 
+    static async includeLocalSSHConfig(scopeName: string, configContent: string): Promise<boolean> {
+        const render = Handlebars.compile(`## START GITPOD {{scopeName}}
+### This section is managed by Gitpod. Any manual changes will be lost.
+
+{{{configContent}}}
+
+## END GITPOD {{scopeName}}`);
+        const newContent = render({ scopeName, configContent });
```

Comment:
No need `Handlebars`, template strings  ``` `${value}` ``` is enough

Reply:
Also we should create a `gitpod` folder and put our config there, in the main config we just add `Include "gitpod/config"`

Reply:
I tried Include, but it's not working https://github.com/gitpod-io/gitpod-vscode-desktop/pull/57/commits/1f143b84a1c677be80156f179d31aff91cecb204

Reply:
I added it seems to be working on my machine :crossed_fingers: